packages/manager/modules/preloader/src/constants.js

```diff
@@ -0,0 +1,26 @@
+export const WELCOME_MESSAGES = [
+  ['Welcome to the ', 'OVHcloud Control Panel'],
+  ['Bienvenue sur votre ', 'Espace client OVHcloud'],
+  ['Benvenuto nel tuo ', 'Spazio Cliente OVHcloud'],
+  ['Herzlich willkommen in Ihrem', 'OVHcloud Kundencenter'],
+  ['Bienvenido/a al ', 'Área de cliente de OVHcloud'],
+  ['Witamy w', 'Panel klienta OVHcloud'],
+  ['Bem-vindo(a) à', 'Área de Cliente OVHcloud'],
+];
+
+export const LOADING_MESSAGES = [
+  'Loading…',
+  'Chargement en cours…',
+  'Caricamento in corso…',
+  'Wird geladen…',
+  'Carregamento em curso…',
+  'Trwa pobieranie informacji...',
+];
+
+export const SWITCH_PARAMETER = 'm';
```

Comment:
maybe something more explicit than m ? ^^

Reply:
Yes, but in fact we don't need to say "Hello we were in web and now we are loading dedicated", `m` parameter should be present in the url (with no value if you want) just to say "I'm coming from another manager, so please don't welcome me again".
Could be used like this: `https://host/path/?m&query=param`